app/feature/travel-certificate/src/main/kotlin/com/hedvig/android/feature/travelcertificate/data/DownloadTravelCertificateUseCase.kt

```diff
@@ -33,7 +33,7 @@ internal class DownloadTravelCertificateUseCase(
           .build()
 
         val downloadedFile = File(
-          context.getExternalFilesDir(Environment.DIRECTORY_DOWNLOADS),
+          context.filesDir,
```

Comment:
In the other place we handle files we're doing this and it was working okay, what do we think is different in this case which breaks it?

Also the file name is of the date today,  Iwas thinking if they try to download two files in one day, one will overwrite the other, should we maybe add also the hour-minute-second to the file name too?

Reply:
Good point.. Im not sure if this fixes the problem to be honest. It seems to work on dev builds so it makes it quite hard to reproduce.

I added the current time to the file path.